### PR TITLE
respect FIFTYONE_DEFAULT_APP_PORT env var

### PR DESCRIPTION
**Pull request by sashankaryal**
respect FIFTYONE_DEFAULT_APP_PORT env var

### CODE REVIEW

File: app/packages/core/src/components/Setup.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ const Code = styled.pre`
 
 const port = (() => {
   if (isElectron()) {
-    return parseInt(process.env.FIFTYONE_SERVER_PORT) || 5151;
```
Comment (benjaminpkane) [inline app/packages/core/src/components/Setup.tsx:50]: This is used by the Desktop App and should be kept (with priority)
* https://github.com/voxel51/fiftyone/blob/10cf8844dee0a9cea716c745916d80ed901074ec/fiftyone/core/service.py#L415

File: app/packages/utilities/src/fetch.ts [hunk 2]
```diff
@@ -178,7 +178,7 @@ export const getAPI = () => {
   }
   return isElectron()
     ? `http://${process.env.FIFTYONE_SERVER_ADDRESS || "localhost"}:${
-        process.env.FIFTYONE_SERVER_PORT || 5151
+        process.env.FIFTYONE_DEFAULT_APP_PORT || 5151
```
Comment (benjaminpkane) [inline app/packages/utilities/src/fetch.ts:181]: Same comment

File: app/packages/core/src/components/Setup.tsx (hunk 1 above)
Comment (sashankaryal) [reply app/packages/core/src/components/Setup.tsx]: Don't know why we're using a different env var for electron apps, but I bet there's a reason. Addressed in next commit.

**Review by benjaminpkane — APPROVED** [review]
LGTM

**Pull request merged**